### ISSUE TITLE
Use of uninitialized value $_ 

### ISSUE THREAD

**Issue by timoxley** [so-fancy/diff-so-fancy#198]
Likely related to #35.

At the end of a `git add --patch` I always get a line like:

`Use of uninitialized value $_ in print at /usr/local/Cellar/git/2.10.0/libexec/git-core/git-add--interactive line 1368, <STDIN> line 7.`

e.g.
![image](https://cloud.githubusercontent.com/assets/43438/18526401/0aba436a-7af3-11e6-8249-db150b011904.png)

I currently have git configured like so:

``` ini
[interactive]
  diffFilter = diff-so-fancy --patch | less --tabs=4 -RFX

[pager]
  diff = diff-so-fancy | less --tabs=4 -RFX
  show = diff-so-fancy | less --tabs=4 -RFX
```

Not a major problem but is a nuisance.

**Comment by scottchiefbaker** [comment]
This is a known issue that we're attempting to fix with #35. Officially d-s-f does not work with `--patch`... it kind of works, but there is an issue with line counts in Git that we can't really fix.

**Comment by scottchiefbaker** [comment]
I'm going to mark this closed for now, as it's part of #35. I'm not sure if we'll be able to address that yet or not. That's an internal `git` issue.
